azure-pipelines.yml

```diff
@@ -21,7 +21,7 @@ stages:
               rustup_toolchain: stable
             linux-pinned:
               imageName: 'ubuntu-16.04'
-              rustup_toolchain: 1.41.0
+              rustup_toolchain: 1.45.2
```

Comment:
do you need the latest version?

Reply:
Probably not, but something in between 1.42 and 1.45.2. Do you want me to check in which version the problem is fixed ?

Reply:
```
error[E0658]: use of unstable library feature 'str_strip': newly added
  --> src/cmd/init.rs:65:26
   |
65 |     match path_to_refine.strip_prefix(LOCAL_UNC) {
   |                          ^^^^^^^^^^^^
   |
   = note: see issue #67302 <https://github.com/rust-lang/rust/issues/67302> for more information

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: could not compile `zola`.
```
This was stabilized in 1.45, (https://github.com/rust-lang/rust/blob/master/RELEASES.md#stabilized-apis-1), I manage to confirm using rustc as well. So minimum version to bump is 1.45 to avoid this error. I'm gonna change that accordingly.